media/css/mozorg/contribute/contribute.scss

```diff
@@ -2,176 +2,396 @@
+    background-image: url("/media/img/contribute/contribute-gethelp-banner-bg.jpg");
+    background-size: cover;
+    color: $color-white;
 
-.contribute-mozillian-list {
-    margin-top: $layout-lg;
+    @media #{$mq-lg} {
+        margin-bottom: $layout-lg;
+    }
 
-    .contribute-mozillian-item {
-        @include border-box;
-        @include text-body-lg;
-        margin-bottom: $spacing-lg;
-        padding: 0 $spacing-md;
+    .contribute-banner-gethelp-title {
+        font-family: Inter, X-LocaleSpecific, sans-serif;
+        @include font-size(24px);
         position: relative;
-        text-align: center;
+
+        @media #{$mq-lg} {
+            @include font-size(40px);
+        }
+
+        &:before {
+            content: "";
+            display: block;
+            background-image: url("/media/img/contribute/contribute-banner-gethelp-icon.svg");
+            position: absolute;
+            height: 40px;
+            width: 40px;
+            left: 0;
+            top: -65px;
+
+            @media #{$mq-lg} {
+                top: auto;
+                left: -80px;
+            }
+        }
+    }
+}
+
+.contribute-banner-problems {
+    background-image: url("/media/img/contribute/contribute-problems-bulb.svg");
+    background-size: 60px 150px;
+    background-position: 10% -10px;
+    background-repeat: no-repeat;
+
+    @media #{$mq-sm} {
+        background-size: 50px 150px;
+        background-position: 8% -10px;
     }
 
     @media #{$mq-lg} {
-        @include clearfix;
+        border: 4px solid $color-moz-light-gray;
+        background-image: url("/media/img/contribute/contribute-problems-bulb.svg"),
+            url("/media/img/contribute/contribute-problems-fly.svg");
+        background-size: 86px 241px, 33%;
+        background-position: 10% -45px, 96% 88%;
+        background-repeat: no-repeat;
+    }
+
+    .contribute-banner-problems-title {
+        @include font-size(24px);
+        margin-left: $layout-xl;
+
+        span {
+            display: inline;
+            background-color: #ffc900;
+            box-shadow: 10px 0 0 #ffc900, -10px 0 0 #ffc900;
+            -webkit-box-decoration-break: clone;
+            -ms-box-decoration-break: clone;
+            -o-box-decoration-break: clone;
+            box-decoration-break: clone;
+        }
+
+        @media #{$mq-lg} {
+            margin-left: 0;
+            @include font-size(40px);
+        }
 
-        .contribute-mozillian-item {
-            width: 33.3%;
-            @include bidi((
-                (float, left, right),
-            ));
+        .contribute-banner-problems-wrapper {
+            margin-bottom: $spacing-md;
         }
     }
 }
 
-.contribute-photo {
-    border-radius: 100%;
-    display: block;
-    height: 180px;
-    margin: 0 auto $spacing-lg;
-    width: 180px;
+.newsletter-subheading {
+    font-weight: bold;
+    font-family: "Zilla Slab", Inter, X-LocaleSpecific, sans-serif;
+    @include font-size(24px);
```

Comment:
Unfortunately it looks like the @include text-display-md mixin renders the font at 40px and 2.5rem instead of the 24px that is outlined in the design.